gitHooks/installHooks.py

```diff
@@ -0,0 +1,5 @@
+#!/bin/python
+
+import subprocess
```

Comment:
I don't get why we need bat,sh **and** python :) ?

Reply:
I simply don't get in which cases you dont have a terminal?

Reply:
After talking to @tsaglam : The clean way to go is just to remove all the installHooks scripts and put it into the development wiki :) We would prefer this way :)

Reply:
Yes :) Otherwise, we have to maintain three files.